packages/react-native/src/primitives/Radio/styles.ts

```diff
@@ -0,0 +1,52 @@
+import { StyleSheet } from 'react-native';
+
+import { RadioStyles } from './types';
+
+export const styles: RadioStyles = StyleSheet.create({
+  container: {
+    alignItems: 'center',
+  },
+  radio: {
+    height: 30,
+    width: 30,
+    borderRadius: 30,
+    borderWidth: 2,
+    padding: 5,
+    alignItems: 'center',
+    justifyContent: 'center',
+    borderColor: '#838c95',
+
+    large: {
+      height: 40,
+      width: 40,
+      borderRadius: 40,
```

Comment:
I'm wondering if it'd be easier to just set an arbitrarily large `borderRadius` in the main `radio` so that it is always a circle regardless of its size. Same goes for `radioButton`.